bbb-common-web/src/main/java/org/bigbluebutton/presentation/UploadedPresentation.java

```diff
@@ -52,6 +54,29 @@ public UploadedPresentation(String podId,
     this.podId = podId;
     this.meetingId = meetingId;
     this.id = id;
+    this.temporaryPresentationId = temporaryPresentationId;
+    this.name = name;
+    this.baseUrl = baseUrl;
+    this.isDownloadable = false;
+    this.current = current;
+    this.authzToken = authzToken;
+    this.uploadFailed = uploadFailed;
+    this.uploadFailReason = uploadFailReason;
+  }
+
+  public UploadedPresentation(String podId,
+                              String meetingId,
+                              String id,
+                              String name,
+                              String baseUrl,
+                              Boolean current,
+                              String authzToken,
+                              Boolean uploadFailed,
+                              ArrayList<String> uploadFailReason) {
+    this.podId = podId;
+    this.meetingId = meetingId;
+    this.id = id;
+    this.temporaryPresentationId = "";
     this.name = name;
     this.baseUrl = baseUrl;
     this.isDownloadable = false;

```

Comment:
Maybe you could call the other constructor instead of repeat all the attributions. 
`this(podId, meetingId, id, "", name, baseUrl, current, authzToken, uploadFailed, uploadFailReason);`